docs/component/signature.md

```
@@ -12,76 +19,106 @@ const img = ref({
+|-----------------|----------------------------------------------------------------------|---------|--------|----------|----------|
+| penColor        | 签名笔颜色                                                           | String  | --     | #000000  | --       |
+| lineWidth       | 签名笔宽度                                                           | Number  | --     | 2        | --       |
+| height          | 画布的高度                                                           | Number  | --     | 200      | --       |
+| width           | 画布的宽度                                                           | Number  | --     | 300      | --       |
+| clearText       | 清空按钮的文本                                                       | String  | --     | 清空     | --       |
+| confirmText     | 确认按钮的文本                                                       | String  | --     | 确认     | --       |
+| fileType        | 目标文件的类型，[wx.canvasToTempFilePath属性介绍](https://developers.weixin.qq.com/miniprogram/dev/api/canvas/wx.canvasToTempFilePath.html#%E5%8F%82%E6%95%B0) | String  | --     | png      | --       |
+| quality         | 目标文件的类型，[wx.canvasToTempFilePath属性介绍](https://developers.weixin.qq.com/miniprogram/dev/api/canvas/wx.canvasToTempFilePath.html#%E5%8F%82%E6%95%B0) | Number  | --     | 1        | --       |
+| exportScale     | 导出图片的缩放比例                                                   | Number  | --     | 1        | --       |
+| disabled        | 是否禁用签名板                                                       | Boolean | --     | false    | --       |
+| backgroundColor | 画板的背景色                                                         | String  | --     | --       |
```

Markdownlint (0.37.0)

Expected: 6; Actual: 5; Too few cells, row will be missing data
Table column count

(MD056, table-column-count)